hw/opentitan/ot_otp_eg.c

```diff
@@ -387,16 +388,22 @@ REG32(LC_STATE, 2008u)
 #define LC_TRANSITION_CNT_SIZE                               48u
 #define LC_STATE_SIZE                                        40u
 
-#define INTR_MASK (INTR_OTP_OPERATION_DONE_MASK | INTR_OTP_ERROR_MASK)
-#define ALERT_TEST_MASK \
+#define INTR_WMASK (INTR_OTP_OPERATION_DONE_MASK | INTR_OTP_ERROR_MASK)
+
+#define ALERT_WMASK \
     (ALERT_FATAL_MACRO_ERROR_MASK | ALERT_FATAL_CHECK_ERROR_MASK | \
      ALERT_FATAL_BUS_INTEG_ERROR_MASK | ALERT_FATAL_PRIM_OTP_ALERT_MASK | \
      ALERT_RECOV_PRIM_OTP_ALERT_MASK)
 
-#define SW_CFG_WINDOW      0x800u
-#define SW_CFG_WINDOW_SIZE (NUM_SW_CFG_WINDOW_WORDS * sizeof(uint32_t))
+#define DIRECT_ACCESS_CMD_WMASK \
+    (R_DIRECT_ACCESS_CMD_RD_MASK | R_DIRECT_ACCESS_CMD_WR_MASK | \
+     R_DIRECT_ACCESS_CMD_DIGEST_MASK | R_DIRECT_ACCESS_CMD_ZEROIZE_MASK)
 
-#define DAI_DELAY_NS 100000u /* 100us */
+#define CHECK_TRIGGER_WMASK \
+    (R_CHECK_TRIGGER_INTEGRITY_MASK | R_CHECK_TRIGGER_CONSISTENCY_MASK)
+
+#define SW_CFG_WINDOW      0x4000u
```

Comment:
This (I presume copy/paste from Darjeeling) is the source of the CI regression failures:
```suggestion
#define SW_CFG_WINDOW      0x800u
```
Earlgrey's `SW_CFG_WINDOW` is it an offset of `0x800` instead of `0x4000`.